test/input/beamville/beam.conf

```diff
@@ -76,7 +76,12 @@ beam.outputs.explodeEventsIntoFiles = false
 # Logging Levels:
 # >0 enables loggin, <=0 suppresses logging.
 # Any event types not explicitly listed take on Default.level
-beam.outputs.defaultLoggingLevel = 1
+# if defaultLoggingLevel is blank then it will treat as OFF.
+beam.outputs.defaultLoggingLevel = "SHORT" # valid options:VERBOSE,REGULAR,SHORT,OFF
+beam.outputs.VERBOSELoggingLevel = ["time","type","person","mode","vehical","departure_time","link"]
```

Comment:
this is better than before, but still needs to be revised. the problem is that the set of fields is different for each type of event and we will have 10-20 event types to specify, so I think it would be too much clutter to do this in the config file. rather we might do this in a class somewhere or from data.